### DISCUSSION TITLE
Unexpected behaviour by Optimizer

### DISCUSSION THREAD

**Discussion by SudoWodo** [SudoWodo/StochOptim#14] [Q&A]
Can there be a scenario where optimizer straight up crashes or shows unexpected behavior for some inputs?  I have relatively less experience with optimization tools and procedures so I don't know.  If there exist such a scenario then how global and local wrappers are expected handle it? If demonstration of such behavior is possible that would be very helpful.

**Comment by nashjc** [comment]
Send us a script so we can try to reproduce the problem. That is the R way!
